docs/source/decisions/0001-api-versioning.rst

```diff
@@ -0,0 +1,53 @@
+############################
+API Versioning and Reporting
+############################
+
+Summary
+=======
+
+API should provide a reason version and list/matrix of capabilities for a given instance of CDA
+
+Opinions
+========
+
+Opinion 1 Calendar versioning
+-----------------------------
+
+@MikeNeilson
+
+Summary: Calendar versioning is easier to support and automate with this API
+
+As the one whose has been making the "official" releases Semantic Version versioning has basically been useless.
+We have been making so many feature additions that if I was doing it right we'd never have a minor version change between releases.
+It's also caused me to, I think, release too slowly.
+
+While we can automate SemVer it is an additional step.
+
+With Calendar Versioning automation tools can just pick the current date when appropriately triggered,
+perhaps by merged into a particular branch.
+
```

Comment:
```suggestion

@adamkorynta 

 Summary: Calendar versioning more directly aligns with consumers needs
 
Deprecation, desupport, and removal notices usually involve timelines rather than predicting x number of major/minor releases into the future.
 
Given the endpoints themselves are already versioned, following semver becomes obtuse with only a real use-case of structural changes such as supported JDK changes, etc. 
```